Source/WebCore/dom/PointerEvent.cpp

```diff
@@ -68,7 +68,12 @@ RefPtr<PointerEvent> PointerEvent::create(MouseButton button, const MouseEvent&
 
 Ref<PointerEvent> PointerEvent::create(const AtomString& type, MouseButton button, const MouseEvent& mouseEvent, PointerID pointerId, const String& pointerType)
 {
-    return adoptRef(*new PointerEvent(type, button, mouseEvent, pointerId, pointerType));
+    return create(type, button, mouseEvent, pointerId, pointerType, typeCanBubble(type), typeIsCancelable(type));
+}
+
+Ref<PointerEvent> PointerEvent::create(const AtomString& type, MouseButton button, const MouseEvent& mouseEvent, PointerID pointerId, const String& pointerType, CanBubble canBubble, IsCancelable isCancelable)
+{
+    return adoptRef(*new PointerEvent(type, button, mouseEvent, pointerId, pointerType, canBubble, isCancelable));
 }
```

Comment:
These changes (while reasonable) seem unrelated to the `getCoalescedEvents` implementation. Can you add an explanation in the commit message?